src/appleseed/foundation/image/oiioimagefilewriter.h

```diff
@@ -0,0 +1,90 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
+// Copyright (c) 2014-2018 Francois Beaune, The appleseedhq Organization
```

Comment:
New file, there should be a single copyright notice dated 2018 and with your name:

    Copyright (c) 2018 Thomas Manceau, The appleseedhq Organization

Reply:
done